src/COLVARS/colvarproxy_lammps_version.h

```diff
@@ -1,3 +1,3 @@
 #ifndef COLVARPROXY_VERSION
-#define COLVARPROXY_VERSION "2025-03-31"
+#define COLVARPROXY_VERSION "2025-05-12"
```

Comment:
@giacomofiorin please remind me: what determines the version of the proxy versus the version of colvars?